backend/docs/swagger.yaml

```diff
@@ -1,12 +1,69 @@
 openapi: 3.0.0
 info:
   description: Backend routes for CSElectives 2.0
-  version: 1.1.0
+  version: 1.2.0
   title: CSElectives API Routes
   contact:
     email: joel.huang@csesoc.org.au
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
```

Comment:
Should link the license to this https://github.com/csesoc/cselectives-v2/blob/develop/LICENSE

Reply:
> Ok! Would you like the name to be "CSElectives 2.0" or "CSElectives v2" or "Uni-lectives"?

CSElectives 2.0